package.json

```diff
@@ -38,13 +38,15 @@
   },
   "homepage": "https://github.com/VandyHacks/vaken#readme",
   "dependencies": {
+    "@types/simple-oauth2": "^2.2.1",
```

Comment:
I suggest moving this to devDependencies (`npm i --save-dev @types/simple-oauth2`) or just move it under `"devDependencies": {}` below